### ISSUE TITLE
Watcher exports unintended figures because of recursion

### ISSUE THREAD

**Issue by silaswaxter** [silaswaxter/inkscape-figure-manager#1]
# TLDR
Currently, on `create`, `edit`, or `watch` a recursive watcher is spawned. This can be undesirable. 

## Example:
You are using InkFigMan to manage some figures for a temporary document in your `$HOME` directory. You are simultaneously working on an unrelated project that has some `.svg` files; maybe you are a graphic designer or some other software works with `.svg` files. All `.svg` files are exported on modification regardless of whether they "should" be from a user perspective.

## Potential Fixes:
-  only spawn recursive watchers if within a repository
-  confirm export
-  enforce that figures are placed in a figures directory
